controllers/farm.controller.js

```diff
@@ -0,0 +1,21 @@
+import Farm from '../models/farm.model';
+
+export const getAllFarms = (req, res) => {
+  Farm.find({}, function (err, farms) {
+    res.send(farms);
+  })
+};
+
+export const getFarmsByZip = (req, res) => {
+  Farm.find({Location_Zip: req.params.zipcode}, (err, farms) => {
+    res.send(farms);
+    console.log(`found ${farms.length} farms`);
```

Comment:
I like to use [debug](https://github.com/visionmedia/debug) for logging, because then you have more control and can remove "noise"